### DISCUSSION TITLE
Cannot insert image into Markdown file

### DISCUSSION THREAD

**Discussion by mjdNjhNJ** [zed-industries/zed#16674] [Feature Requests]
I have a simple Markdown syntax in the file:
```
# Representative_1

![](./Media/Appearance_1.jpg)
```

When I click on the preview button it doesn't render the image:
<img width="451" alt="image" src="https://github.com/user-attachments/assets/cb379554-16dc-4c22-a72b-a7da89f49962">

The issue is reproducible with any image. Image that has 831 KB with 1406 × 1404 resolution. Image that has 232 KB with 1024 × 813 resolution. Or even PNG that has 6KB with 514 × 308 resolution. It doesn't render even if I try to use Zoom In button.

**Comment by bajrangCoder** [comment]
That's currently unavailable but there is a pr for it
Check : https://github.com/zed-industries/zed/pull/16192
